server/routerlicious/packages/services/src/dbFactory.ts

```diff
@@ -0,0 +1,55 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IDbFactory } from "@fluidframework/server-services-core";
+import debug from "debug";
+import { Provider } from "nconf";
+import * as services from "./";
+
+interface IDBFactoryConfig {
+    name: string;
+    modulePathOrName: string;
+    dbFactoryConstructorName: string;
+    dbConfig: any;
+}
+
+const creator = (dbFactoryConfig: IDBFactoryConfig) => async (): Promise<IDbFactory> => {
+    debug(`Using ${dbFactoryConfig.name} Database`);
```

Comment:
@znewton @pradeepvairamani do you know if we get to call `dbFactory` from client code? Since this `dbFactory` is placed under the services package, I think we can assume it's Node.js exclusively, right?

If that is the case, can we use `Lumberjack` instead of `debug`?
More info about Lumberjack available here: https://github.com/microsoft/FluidFramework/blob/main/server/routerlicious/packages/services-telemetry/README.md

Reply:
Correct yes. I think `debug` is a relic here and probably outside the scope of this change. But bonus kudos to @nedalhy if you wanna change it!

Reply:
Yes sure, I can change it.